<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncluirCD</class>
 <widget class="QDialog" name="IncluirCD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Incluir CD</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Downloads/DVD1.png</normaloff>../../../Downloads/DVD1.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox_CD">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>190</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>INCLUIR NOVO CD</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Preço:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nº  Faixas:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nome_CD">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>180</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="preco_CD">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>180</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="faixas_CD">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>180</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_CD</sender>
   <signal>accepted()</signal>
   <receiver>IncluirCD</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_CD</sender>
   <signal>rejected()</signal>
   <receiver>IncluirCD</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
